pages/opam-using.html

```diff
@@ -70,4 +87,64 @@ <h2>Using OPAM to install Coq packages</h2>
 
 <pre><code>opam install coq-sudoku
 </code></pre>
+
+<h2>Advanced usage</h2>
+
+<h3>Managing different versions of OCaml and Coq</h3>
```

Comment:
What about merging this title with the one above, so that `<h3>` can be used for switches and roots? The Coq website style for `<h4>` HTML tags is a tad disappointing.